cli_reference/tkn_cli/op-tkn-reference.adoc

```diff
@@ -0,0 +1,42 @@
+[id='op-tkn-reference']
+= Pipeline `tkn` reference
+include::modules/common-attributes.adoc[]
+include::modules/openshift-pipelines-attributes.adoc[]
+:context: op-tkn-reference
+toc::[]
+
+
+This section lists basic `tkn` CLI commands. For a more detailed description, see link:https://github.com/tektoncd/cli[Tekton Pipelines CLI].
+
+== Basic Syntax
+`tkn [command or options] [arguments...]`
+
+== Global Options
+`--help, -h`
+
+// Utility commands
+include::modules/op-tkn-utility-commands.adoc[leveloffset=+1]
+
+// Pipeline management commands
+include::modules/op-tkn-pipeline-management.adoc[leveloffset=+1]
+
+// Pipeline run commands
+include::modules/op-tkn-pipeline-run.adoc[leveloffset=+1]
+
+// Task management commands
+include::modules/op-tkn-task-management.adoc[leveloffset=+1]
+
+// Task run commands
+include::modules/op-tkn-task-run.adoc[leveloffset=+1]
+
+// Condition management commands
+include::modules/op-tkn-condition-management.adoc[leveloffset=+1]
+
+// Pipeline resources commands
+include::modules/op-tkn-pipeline-resource-management.adoc[leveloffset=+1]
+
+// Cluster Task management commands
```

Comment:
I only now realized that it's a comment but still... `Cluster Task`

Reply:
Since it won't appear in customer-facing document, I'm OK with ignoring it.

Reply:
Ok ,Thanks @ppitonak. Could you please provide an ACK/lgtm